tempus_fugit_minecraft/player.py

```diff
@@ -291,3 +291,25 @@ def keep_player_within_coordinates(dimension, boundary_size=WORLD_SIZE):
             return -boundary_size
         else:
             return dimension
+
+    def slow_walk(self) -> None:
+        """!
+        @brief Slows the player's walking speed
+        @see [Issue#97](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/97)
+        """
+        self.walking_speed = self.walking_speed / 3
```

Comment:
Probably something that needs to be done in a different pull request, but we probably have to change walking_speed to be more specific and include units.